mlperf_logging/benchmark_meta.py

```diff
@@ -15,6 +15,7 @@
         'ncf': 10,
         'rnnt': 10,
         'unet3d': 40,
+        'llama2_70b_lora': 10,
```

Comment:
I'm not convinced this is a good benchmark name, seems a bit overcomplicated. Have we agreed on this? How about just `lora`?